package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./www"
+    "start": "node ./www",
+    "seed": "node ./lib/server/db/seeds/seed.js"
```

Comment:
Should add a `"test"` script that calls `jasmine`, which will read that config below and run the specs specified.